package.json

```diff
@@ -34,6 +34,8 @@
     "express": "^4.17.1",
     "express-healthcheck": "^0.1.0",
     "feedparser-promised": "^2.0.1",
+    "sentiment": "^5.0.2",
+    "wordcount": "^1.1.1"
```

Comment:
This will break, you are missing a `,` here

Reply:
Sorry, my fault. I have updated `package.json` now and rebased.